core.js

```diff
@@ -477,6 +477,7 @@ class FileTypeParser {
 			const brandMajor = this.buffer.toString('binary', 8, 12).replace('\0', ' ').trim();
 			switch (brandMajor) {
 				case 'avif':
+				case 'avis':
```

Comment:
Are you sure this is right extension and MIME-type for AVIF Sequence file?
What about `.avifs` and `image/avif-sequence`

Ref:
- http://fileformats.archiveteam.org/wiki/AVIF
- https://github.com/AOMediaCodec/libavif/issues/112)?

Reply:
Never mind, found the answer: https://github.com/AOMediaCodec/av1-avif/issues/59#issuecomment-645401286